src/test/java/learn/field_agent/data/AliasJdbcTemplateRepositoryTest.java

```diff
@@ -69,8 +69,8 @@ void shouldNotUpdateNonexistent() {
 
     @Test
     void shouldDelete() {
-        assertTrue(repository.deleteById(1));
-        assertFalse(repository.deleteById(1));
+        assertTrue(repository.deleteById(3));
+        assertFalse(repository.deleteById(3));
```

Comment:
Deleting alias ID `1` was interfering with your agent unit test that was checking that an agent's aliases were being loaded.